.npmrc

```diff
@@ -2,4 +2,6 @@
 # packages as it casues issues with manypkg
 # @link https://pnpm.io/npmrc#prefer-workspace-packages
 save-workspace-protocol=false
-prefer-workspace-packages=true
\ No newline at end of file
+prefer-workspace-packages=true
+
```

Comment:
nit: extra space

Reply:
It was intentional to separate it from the other 2 that relates to the comment

Reply:
Ah fair. All good 👍 I mentally skip over comments😅